client/src/components/Header.js

```diff
@@ -4,13 +4,20 @@ import styled from "styled-components";
 // import Container from "@mui/material/Container";
 import "../App.css";
 
-const StyledContainer = styled(Container)`
- h1 {
-    font-size: 1.5em;
-    text-align: center;
+const Container = styled.h1`
+ 
+  font-size: 1.5em;
+  text-align: center;
   }
 `;
 
+// const Container = styled.container`
+//   h1 {
+//     font-size: 1.5em;
+//     text-align: center;
+//   }
+// `;
+
```

Comment:
This looks like a regression.  Did you mean to re-introduce this code?  Assuming not, would you please fix it before merging?